.gitignore

```diff
@@ -1,16 +1,17 @@
-# Directories/files that may be generated by this project
-build
-build-module
-build-style
-node_modules
-gutenberg.zip
-languages/gutenberg.pot
-/languages/gutenberg-translations.php
-
-# Directories/files that may appear in your environment
-.DS_Store
-*.log
-phpcs.xml
-yarn.lock
-docker-compose.override.yml
-/wordpress
+# Directories/files that may be generated by this project
+build
+build-module
+build-style
+node_modules
+gutenberg.zip
+languages/gutenberg.pot
+/languages/gutenberg-translations.php
+
+# Directories/files that may appear in your environment
+.DS_Store
+*.log
+phpcs.xml
+yarn.lock
+docker-compose.override.yml
+/wordpress
+/nbproject/*
```

Comment:
Any reason for the change here?

Reply:
I'm using netbeans and this IDE puts its project configurations in this folder directly in the project. For me it is useful to if I want to branch / swicth easily without commit this folder.

Reply:
I see, in that case, I suggest using the global gitignore instead of adding this to all the projects you work on https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Hum, thanks I'm doing and testing it.
It's useful for all my projects :-)

Reply:
It's done. It works and I also removed this folder from .gitignore file

Reply:
Can you restore the .gitignore file to its original version (it shouldn't appear in the changes)

Reply:
Done ! It's due to line feed. Very strange because editorconfig is well configured.